libs/components/src/lib/side-sheet/stories/vl-side-sheet.stories-arg.ts

```diff
@@ -1,50 +1,96 @@
-export const sideSheetArgs = {
+import { Args, ArgTypes } from '@storybook/web-components';
+import { TYPES } from '@domg-wc/common-utilities';
+
+export const sideSheetArgs: Args = {
```

Comment:
Kan je het type 'Args' hier weglaten? Door dit type te plaatsen verlies je je typing in de stories.